qiling/os/windows/dlls/wsock32.py

```diff
@@ -72,3 +72,29 @@ def hook_connect(ql, address, params):
         sep="",
     )
     return 0
+
+#hostent * gethostbyname(
+#  const char *name
+#);
+#typedef struct hostent {
+#  char  *h_name;
+#  char  **h_aliases;
+#  short h_addrtype;
+#  short h_length;
+#  char  **h_addr_list;
+#} HOSTENT, *PHOSTENT, *LPHOSTENT;
+@winapi(cc=STDCALL, params={
+    "name": STRING
+})
+def hook_gethostbyname(ql, address, params):
+    dummy_ip = b"\x40\x30\x20\x10" #10.20.30.40
```

Comment:
we should store this in profile? its in qiling/profile/windows.ql

Reply:
Thanks! Do you want me to just add a [Network] section?

Reply:
sure, any appropriate name will work

Reply:
Added as [NETWORK]: dns_response_ip